### PR TITLE
{bio} BFAST 0.7.0a foss-2016b with patch for inline functions

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/BFAST/BFAST-0.7.0a-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+verdir = ''.join(char for char in version if not char.isalpha())
+source_urls = ['http://sourceforge.net/projects/bfast/files/bfast/%s/' % verdir, 'download']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ed8de49693165a87d5dbef352207c424b1bf6f670a83acf49a4f4f188444995e']
+
+dependencies = [('bzip2', '1.0.6')]
+
+patches = ['BFAST-0.7.0a-inline.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/b/BFAST/BFAST-0.7.0a-foss-2016b.eb]: @verdurin please avoid hardcoding version twice, use `%(version)s` here:

```python
patches = ['BFAST-%(version)s-inline.patch']
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2451.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/73d05537a1a275414d0604ca28847232 for a full test report.

File: easybuild/easyconfigs/b/BFAST/BFAST-0.7.0a-foss-2016b.eb (hunk 1 above)
Comment (verdurin) [inline easybuild/easyconfigs/b/BFAST/BFAST-0.7.0a-foss-2016b.eb]: @boegel yep, fixed

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2695.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a101b27fb1eec1cdfc71853bba059bf1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**